tools/BuildAssets/targets/common.Build.props

```diff
@@ -14,7 +14,7 @@
 		<DelaySign>true</DelaySign>
 		<AssemblyOriginatorKeyFile>$(LibraryToolsFolder)\MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
 		<NoWarn>1591;1701;1573;CS1591</NoWarn>
-		<DefaultPathTokenToIgnore>$(IgnorePathTokens) Microsoft.Azure.KeyVault.Samples</DefaultPathTokenToIgnore>
+		<DefaultPathTokenToIgnore>$(DefaultPathTokenToIgnore) $(IgnorePathTokens)</DefaultPathTokenToIgnore>
```

Comment:
@shahabhijeet I removed Microsoft.Azure.KeyVault.Samples from this list because I believe you told me it is no longer needed if that isn't the case or I misunderstood I can keep it here.